src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list list">
+          <li class="list__item">
+            <a href="#" class="list__link is-active">Apple</a>
+          </li>
+          <li class="list__item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="list__item">
```